rcp/org.testeditor.rcp4.tcltestrun/src/org/testeditor/rcp4/tcltestrun/TclMavenLauncher.xtend

```diff
@@ -38,22 +42,26 @@ public class TclMavenLauncher implements TclLauncher {
 	@Inject extension ProjectUtils
 	@Inject MavenExecutor mavenExecutor
 
-	override LaunchResult launchTest(List<String> testCases, IProject project, IProgressMonitor monitor, OutputStream out,
-		Map<String, Object> options) {
+	override LaunchResult launchTest(List<String> testCases, IProject project, IProgressMonitor monitor,
+		OutputStream out, Map<String, Object> options) {
 		val parameters = if (options.containsKey(
 				PROFILE)) {
 				"clean generate-test-sources org.testeditor:testeditor-maven-plugin:testEnvUp org.testeditor:testeditor-maven-plugin:testExec -P" +
 					options.get(PROFILE)
 			} else {
 				"clean integration-test"
 			}
-			
+
 		var testSelectionArgument = ""
-		if(testCases != null) {
+		if (testCases != null) {
 			testSelectionArgument = "test=" + testCases.join(",")
-		} 
-		val result = mavenExecutor.executeInNewJvm(parameters, project.location.toOSString,
-			testSelectionArgument, monitor, out, false)
+		}
+		if(!checkMavenVersion(project, monitor)) {
```

Comment:
If you add the –V to the maven call, the test is executed anyway. But we need a sanity check before the test starts.